include/nbl/builtin/glsl/ext/Scan/parameters_struct.glsl

```diff
@@ -0,0 +1,11 @@
+#ifndef _NBL_GLSL_EXT_SCAN_PARAMETERS_STRUCT_INCLUDED_
+#define _NBL_GLSL_EXT_SCAN_PARAMETERS_STRUCT_INCLUDED_
+
+struct nbl_glsl_ext_Scan_Parameters_t
+{
+	uint stride;
+	uint element_count_pass;
+	uint element_count_total;
+};
```

Comment:
Since `element_count_total` is basically a constant among all dispatches, would it make sense to make it a macro instead like
```
#define _NBL_GLSL_WORKGROUP_SIZE_ %u
#define _NBL_GLSL_EXT_SCAN_BIN_OP_ %u
#define _NBL_GLSL_EXT_SCAN_STORAGE_TYPE_ %s
#define _NBL_GLSL_EXT_SCAN_TOTAL_ELEMENT_COUNT_ %u
```
but that would mean you have to recompile your shader everytime input element count changes.

Reply:
bad idea to recompile when input size changes